gax-grpc/src/main/java/com/google/api/gax/grpc/ApiException.java

```diff
@@ -36,7 +36,7 @@
 /**
  * Represents an exception thrown during an RPC call.
  *
- * <p>It stores information useful for functionalities in {@link UnaryCallable} and {@link
+ * <p>It stores information useful for functionalities in {@link UnaryGrpcCallable} and {@link
```

Comment:
`ApiException` is something that will need to move up to the core gax module. (Doesn't need to be in this PR)